README.md

```diff
@@ -40,7 +40,7 @@ sudo apt-get install npm
 ```
 Then to install wio, you have to do:
 ```bash
-npm install -g wio
+sudo npm install -g wio --unsafe-perm
```

Comment:
it isn't good to have to install `wio` in such a sketchy way. You're going to need to find a better way to do this

Reply:
honestly, this is the only way when you write to disk while installing an npm package. With `--unsafe-perm` flag it will allow the write. Optionally we can pack the executables with the build and hence we do not need to pull them but we will see in future

Reply:
Well clearly there are other ways plenty of npm packages build executables during install. Whenever you install sass it pretty much builds from the c++ source lol

Reply:
No building is fine cuz you are using tools on the machine. Wio grabs executables from github versions and pastes them in the directory.

Reply:
Well I'm sure there are other packages that install binaries with `npm` without having to do this flag. You should find a way around it because it kinda makes wio sketch lol